wp-content/themes/pub/wporg-learn-2024/patterns/contribute-page-content.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Title: Contribute Page Content
+ * Slug: wporg-learn-2024/contribute-page-content
+ * Inserter: no
+ */
+
+?>
+ 
+<!-- wp:heading {"style":{"typography":{"fontSize":"36px","fontStyle":"normal","fontWeight":"400"}},"fontFamily":"eb-garamond"} -->
+<h2 class="wp-block-heading has-eb-garamond-font-family" style="font-size:36px;font-style:normal;font-weight:400">
+	<?php esc_html_e( 'Contribute', 'wporg-learn' ); ?>
+</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"},"spacing":{"margin":{"bottom":"40px","top":"0px"}}},"fontSize":"normal","fontFamily":"inter"} -->
+<p class="has-inter-font-family has-normal-font-size" style="margin-top:0px;margin-bottom:40px;font-style:normal;font-weight:400">
+	<?php esc_html_e( "Here's how you can get involved and create content for Learn WordPress", 'wporg-learn' ); ?>
+</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}},"fontSize":"extra-large","fontFamily":"inter"} -->
+<h2 class="wp-block-heading has-inter-font-family has-extra-large-font-size" style="font-style:normal;font-weight:600">
+	<?php esc_html_e( 'Create a Tutorial', 'wporg-learn' ); ?>
+</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"},"spacing":{"margin":{"bottom":"var:preset|spacing|20","top":"var:preset|spacing|20"}}},"fontSize":"small","fontFamily":"inter"} -->
+<p class="has-inter-font-family has-small-font-size" style="margin-top:var(--wp--preset--spacing--20);margin-bottom:var(--wp--preset--spacing--20);font-style:normal;font-weight:400">
+	<?php
+	/* translators: 1: opening a tag for a URL, 2: closing a tag, 3: opening a tag for a URL, 4: closing a tag */
+	printf(
+		esc_html__(
+			'Tutorials can be on any topic related to WordPress, can be for any level of experience, and in any language. 
+			To get started, check out this %1$s tutorial about creating tutorials%2$s. Then %3$s submit your tutorial presenter application%4$s.',
+			'wporg-learn'
+		),
+		'<a href="https://learn.wordpress.org/workshop/how-to-submit-a-workshop/">',
+		'</a>',
+		'<a href="https://learn.wordpress.org/workshop-presenter-application/">',
+		'</a>'
```

Comment:
Close! You can include the tags and just esc the urls
```suggestion
	/* translators: 1: tutorial creation URL, 2: tutorial presenter application URL */
	printf(
		esc_html__(
			'Tutorials can be on any topic related to WordPress, can be for any level of experience, and in any language. 
			To get started, check out this <a href="%1$s">tutorial about creating tutorials</a>. Then <a href="%2$s">submit your tutorial presenter application</a>.',
			'wporg-learn'
		),
		esc_url( 'https://learn.wordpress.org/workshop/how-to-submit-a-workshop/' ),
		esc_url( 'https://learn.wordpress.org/workshop-presenter-application/' ),
```

Reply:
I have thought about that, but won't the translators see the `<a href="%2$s">`?

Reply:
Wait that doesn't include the translation call...

Reply:
Example [here](https://github.com/WordPress/wporg-developer/blob/8b33d50a34a8a539a9f2980c4645d1b7fa5850d0/source/wp-content/themes/wporg-developer-2023/patterns/front-page-header.php#L43)

Reply:
yeah, thanks. It looks like [it's fine](https://translate.wordpress.org/projects/meta/wordpress-org/zh-tw/default/?filters%5Bstatus%5D=current) to have html in the strings as you said.

Reply:
> I have thought about that, but won't the translators see the `<a href="%2$s">`? and is `esc_url` needed here? As I thought the URL is not a parameter from elsewhere.

Yeah you're right escaping the urls is probably not necessary. You could do `site_url( '/workshop-presenter-application/' )` to make it relative to the current site.

Reply:
033b1ee